### PR TITLE
Update TLD Regex Max Length

### PR DESCRIPTION
**Pull request by lucemans**
## What does this PR do?

Changes the Regex validation for profile editing the website field.

Fixes # (issue)

## Type of change

- [ ] Enhancement (non-breaking small changes to existing functionality)

## How should this be tested?

- [ ] All TLDs work now

## Checklist

- I haven't commented my code, particularly in hard-to-understand areas
  (Because it is a few character change)
- I haven't checked if my PR needs changes to the documentation
  (I hope it doesnt lol)
- I haven't checked if my changes generate no new warnings
  (edited through github gui so dont know)
- I haven't added tests that prove my fix is effective or that my feature works
  (... cuz is small)
- I haven't checked if new and existing unit tests pass locally with my changes
  (cuz github ui)

### CODE REVIEW

**Comment by Jontes-Tech** [comment]
LGTM! 👍

File: src/constants.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const DEFAULT_COLLECT_TOKEN = IS_MAINNET
 
 // Regex
 export const URL_REGEX =
-  /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[\da-z]+([.\-][\da-z]+)*\.[a-z]{2,5}(:\d{1,5})?(\/.*)?$/;
+  /^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[\da-z]+([.\-][\da-z]+)*\.[a-z]{2,63}(:\d{1,5})?(\/.*)?$/;
```
Comment (bigint) [inline src/constants.ts:76]: Thanks for the contribution @lucemans 🙏🏼 

**question:** don't 63 sound huge?

Comment (Jontes-Tech) [reply src/constants.ts:76]: It may sound huge, but that's the maximum TLD length according to ICANN.

Comment (bigint) [reply src/constants.ts:76]: Ahh that make sense, just got to know 🙏🏼 Thanks

**Review by bigint — APPROVED** [review]
**LGTM** :shipit: 

Rad LGTM

**Comment by Jontes-Tech** [comment]
Yey! Thanks for the contribution Luc, I'm sure this'll be very useful to owners of domains like luc.computer, luc.contact, etc.

**Comment by bigint** [comment]
Hey fren, you are now eligible for Gitpoap for contributing to Lenster 🌸, go and claim your beautiful POAP 🏃🏼‍♂️ 

https://www.gitpoap.io/gh/lensterxyz/lenster

**Comment by Jontes-Tech** [comment]
What the hell's a poap?

**Comment by bigint** [comment]
@Jontes-Tech It's an on-chain `proof-of-attendance-protocol`

This contributor POAP helps you to show that you are a Lenster contributor and the holders might be eligible for some swags or event tickets 🙇🏼 

You can learn more here https://poap.xyz/

**Pull request merged**